expr/ztests/slice.yaml

```
@@ -0,0 +1,14 @@
+zql: |
+   cut a1=a[1:-1],a2=a[1:],a3=a[:1],a4=a[:-1],a5=a[:-100],a6=a[-1:]
```

```suggestion
   cut a1=a[1:-1],a2=a[1:],a3=a[:1],a4=a[:-1],a5=a[:-100],a6=a[-1:], a7=[:a[0]]
```
(or some other additive expr in the brakcets)